Formula/vsftpd.rb

```diff
@@ -44,11 +44,45 @@ def caveats
       s += <<-EOS.undent
         vsftpd was compiled with SSL support. To use it you must generate a SSL
         certificate and set 'enable_ssl=YES' in your config file.
+
       EOS
     end
 
+    s += <<-EOS.undent
+      vsftpd requires root privileges in most cases so you will need to run
+      `sudo vsftpd`.
+      You should be certain that you trust any software you grant root privileges.
+
+      The vsftpd.conf file must be owned by root in order to work correctly:
+        sudo chown root #{HOMEBREW_PREFIX}/etc/vsftpd.conf
+    EOS
     s
   end
+
+  plist_options :startup => true, :manual => "sudo vsftpd"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>#{sbin}/vsftpd</string>
+        <string>#{etc}/vsftpd.conf</string>
+      </array>
+      <key>RunAtLoad</key>
+      <true/>
+    </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{sbin}/vsftpd -v 0>&1")
```

Comment:
By default, `vsftpd -v` outputs version info to stdin. To test the version properly, stdin must be redirected to stdout (0>&1).